repoqa/provider/embeddings/openai.py

```diff
@@ -0,0 +1,25 @@
+import os
+from typing import List, Tuple
+
+from openai import Client
+from sklearn.metrics.pairwise import cosine_similarity
```

Comment:
sklearn is a bit huge. ideally we want some lighter dependencies (avoiding the risk of dep conflicts in the future) or just impl the function ourselves.